x/stake/tick_test.go

```diff
@@ -89,6 +106,7 @@ func TestProcessProvisions(t *testing.T) {
 	var totalSupply int64 = 550000000
 	var bondedShares int64 = 150000000
 	var unbondedShares int64 = 400000000
+	var provisionTallied int64 = 0
```

Comment:
maybe add a comment, not obvious what this variable's role is directly from the name

Reply:
maybe a better name could be cumulativeExpProvisions based on what I see it's being used for, although that name is a bit long.

Reply:
i'll try to think of one that is shorter than cumulativeExpProvisions, but if not I might just use it

Reply:
fixed